app/build.gradle.kts

```diff
@@ -84,4 +85,13 @@ dependencies {
   implementation(Libs.squidb_annotations)
   implementation(Libs.ink_page_indicator)
   add("kapt", Libs.squidb_processor)
+
+  // Navigation component
+  val navVersion = "2.3.0-rc01"
```

Comment:
We will want to put this in Libs, if you go to Libs.kt you will find the command to update that file, it should automatically add this new one and then you replace your usage here with `Libs.navigation` or whatever it will call it.
Sometimes the command shuffles the file around a bit but I think it has been behaving recently, best to do this as a standalone commit